packages/aws-rfdk/lib/core/lib/log-group-factory.ts

```diff
@@ -64,7 +71,20 @@ export class LogGroupFactory {
         new LogRetention(scope, logWrapperId, {
           logGroupName: fullLogGroupName,
           retention,
+          logRetentionRetryOptions: {
+            base: Duration.millis(200),
+            maxRetries: 7,
+          },
         }).logGroupArn);
+
+    const logRetentionFunctionLogicalId = 'LogRetentionaae0aa3c5b4d4f87b02d85b201efdd8a';
```

Comment:
Same thing here, let's add a short comment saying this is taken directly from the CDK code

Reply:
Will do